### PR TITLE
Minor attempt to clear up the behaviour in the documentation

### PR DESCRIPTION
**Pull request by dolezvo1**
I'm not sure what is the proper terminology for `public` vs `public_03_add_column` schemas. Major/main vs minor/specific? I think there definitely should be a consistent terminology, though.

### CODE REVIEW

File: docs/README.md [hunk 1]
```diff
@@ -572,9 +572,11 @@ The tables and functions in this schema store `pgroll`'s internal state and are
 $ pgroll start sql/03_add_column.json
 ```
 
-This starts the migration defined in the `sql/01_create_table.json` file.
+This starts the migration defined in the `sql/03_add_column.json` file.
 
-After starting a migration there will be two schema versions in the database; one for the old schema before the migration and one for the new version with the schema changes.
+After starting a migration there will be two schema versions in the database; one for the old schema before the migration (e.g. `public_02_create_table`) and one for the new version with the schema changes (e.g. `public_03_add_column`). At this point, the earlier version will be still available under the main schema name (e.g. `public`).
```
Comment (andrew-farries) [inline docs/README.md:577]: I think it's important to make it clear that the versioned schema (like `public_02_create_table`) contain views on the tables in the `public` schema. Applications using schema managed by`pgroll` shouldn't connect directly to `public` but rather use the versioned views.

```suggestion
After starting a migration there will be two schema versions in the database; one for the old schema before the migration (e.g. `public_02_create_table`) and one for the new version with the schema changes (e.g. `public_03_add_column`).  Each schema contains views on the tables in the `public` schema.
```

File: docs/README.md [hunk 2]
```diff
@@ -598,7 +600,7 @@ This completes the most recently started migration.
 
 Running `pgroll complete` when there is no migration in progress is a no-op.
 
-Completing a `pgroll` migration removes the previous schema version from the database, leaving only the latest version of the schema.
+Completing a `pgroll` migration removes the previous schema version from the database (e.g. `public_02_create_table`), leaving only the latest version of the schema (e.g. `public_03_add_column`), available under both specific and main schema name (i.e. both `public_03_add_column` and `public`).
```
Comment (andrew-farries) [inline docs/README.md:603]: Another attempt to make it clear that the version schema contain views and the base schema (default `public`) contains the tables.

```suggestion
Completing a `pgroll` migration removes the previous schema version from the database (e.g. `public_02_create_table`), leaving only the latest version of the schema (e.g. `public_03_add_column`). At this point, any temporary columns and triggers created on the affected tables in the `public` schema will also be cleaned up, leaving the table schema in its final state.
```

**Review by andrew-farries — CHANGES_REQUESTED** [review]
Thanks for adding the extra explanations to the docs 🎉 

I've made some suggested changes to better explain the difference between the versioned schema containing the views and the base schema containing the tables.

**Comment by dolezvo1** [comment]
@andrew-farries Thank you, for your notes, I actually wasn't even completely aware of this. I will apply your changes and try again.

**Review by andrew-farries — APPROVED** [review]

**Comment by dolezvo1** [comment]
Thank you!

**Pull request merged**